packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -293,7 +293,7 @@ export class List extends BaseComponent<IListProps, IListState> {
     let divProps = getNativeProps(this.props, divProperties);
 
     // assign list if no role
-    role = role || 'list';
+    role = (typeof role === 'undefined') ? 'list' : role;
```

Comment:
(We use `=== undefined` check in many places, so I'm wondering if there's something better about the more verbose `typeof thing === 'undefined'` check.)